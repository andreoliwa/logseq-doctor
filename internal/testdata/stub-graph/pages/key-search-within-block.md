- TODO Buy groceries outside
- Parent block
  - Nested item
  - TODO Buy groceries inside
  - Another nested item
- Last item
