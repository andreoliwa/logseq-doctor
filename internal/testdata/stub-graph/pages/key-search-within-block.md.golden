- TODO Buy groceries outside
- Parent block
	- Nested item
	- TODO Clean the room
	- Another nested item
- Last item
